Makefile

```diff
@@ -39,7 +39,7 @@ check: install
 
 test: install
 	@pip install --quiet --requirement=requirements-tests.txt
-	@py.test
+	@py.test --cov=pymodbus/ --cov-report term-missing
```

Comment:
while you're changing this line, you might as well switch to the nowadays recommended `pytest` entry point instead of `py.test`.